tests/campaign.cfg

```diff
@@ -0,0 +1,93 @@
+[common]
```

Comment:
I would simply not include this file and entire folder here.  You've got a functioning version at the test-tool, seems like a better spot.

Reply:
Yes

Reply:
So, could you take these files and folders out of the pull?

Reply:
done